### ISSUE TITLE
Spanish Translation

### ISSUE THREAD

**Issue by Yurian00** [taragnor/city-of-mist#61]
Hi Taragnor:

First of all thank you very much for your amazing work here creating this System for Foundry :-).

I'm interested on translate the system to Spanish to have it available on Foundry so that is the reason I'm using this issue to contact you. My expertise on Foundry is really basic so I can't edit code per se being confident enough to don't screw the whole thing but if you can provide me the texts that need to be translated or tell me which parts of the code need to be I will be pleasure to to it and send it back to you to add Spanish as available language for your system.

Anyways thank you in advance and I hope we can collaborate on this matter.

Best regards,

Yurian.

**Comment by taragnor** [comment]
Sure I'd love to add in a Spanish translation if you'd be willing to make one. 

Doing translations in Foundry VTT is pretty easy. You'll only need to work on a single file, which would be called es.json and go in the lang folder. 

As a template for what needs to be translated: here's the example of English and German localizations.   

Here's the file in English: https://github.com/taragnor/city-of-mist/blob/main/src/city-of-mist/lang/en.json

This is the German localization file: https://github.com/taragnor/city-of-mist/blob/main/src/city-of-mist/lang/de.json

Just follow that same pattern, putting the Spanish Translations on the right hand side. It shouldn't require any coding experience, just copy the format used by the other localizations.

You can send me the file through a github pull request, and I can link it to the rest of the project. 

Let me know if you have any questions.

**Comment by Yurian00** [comment]
Pretty clear. I did not dig enought to check that there was a json with the lanaguages so it is pretty easy. I'll be back as soon as I have it translated.

Thank you very much for your help.

**Comment by Yurian00** [comment]
There are a little diferences on the code side in the json at least in the first lines. I assume it is better if I use the English json as base for the tranlation, right?
![imagen_2024-04-13_005419071](https://github.com/taragnor/city-of-mist/assets/74567425/4eb6b1b8-3b94-40ff-adc3-9f0d386d8cc4)
![imagen_2024-04-13_005448766](https://github.com/taragnor/city-of-mist/assets/74567425/7d8022cc-8e3c-4044-bdf5-d175db59c108)

**Comment by taragnor** [comment]
Yeah not all the translation files are entirely up to date except for English version. I added quite a few new translation keys and the other translators haven't quite caught up (translations usually are a few versions behind). 

 So use the translation keys (the period-separated names on the left hand side) from the en.json for the best results. 

Thanks.

**Comment by Yurian00** [comment]
Nice. I've advance over that file but I see that Oherscape and LitM are there too and I'm not familiarized with them yet so that translation will be more a "guessing" XD.

Additionaly I've detected that with more than 2 spectrums in dangers those overlap with the tabs (movement/tags, themes, bio) as you can see here.
![imagen_2024-04-18_175346426](https://github.com/taragnor/city-of-mist/assets/74567425/b44e9fcf-9c90-4774-bc9a-0976c65506fa)

**Comment by Yurian00** [comment]
It only happens during the edition of the character sheet. As soon as you close it, it looks ok so it is a minor issue.
